codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -108,6 +108,8 @@ data Picture
     | CoordinatePlane CallStack
     | Logo CallStack
     | Pictures [Picture]
+    | Blank Picture
```

Comment:
Update `blank` below to  this?

    blank :: HasCallStack => Picture
    blank = Blank callStack